src/crossover.rs

```diff
@@ -6,37 +6,44 @@
 
 //! Fee module contains the logic related to `Crossover` structure
 
-use dusk_bls12_381::BlsScalar;
 use dusk_bytes::{DeserializableSlice, Error as BytesError, Serializable};
 use dusk_jubjub::{JubJubAffine, JubJubExtended};
-use dusk_poseidon::cipher::PoseidonCipher;
-use dusk_poseidon::sponge;
 
 #[cfg(feature = "rkyv-impl")]
 use rkyv::{Archive, Deserialize, Serialize};
 
+use crate::note::ENCRYPTION_SIZE;
+
 /// Crossover structure containing obfuscated encrypted data
-#[derive(Clone, Copy, Debug, Default)]
```

Comment:
can the `Default` trait not be derived anymore?

Reply:
I had to implement it myself because the new encryption attribute doesn't implements it